policy/modules/system/udev.if

```diff
@@ -320,6 +320,25 @@ interface(`udev_read_runtime_files',`
 	read_lnk_files_pattern($1, udev_runtime_t, udev_runtime_t)
 ')
 
+########################################
+## <summary>
+##	Read udev runtime links.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`udev_read_runtime_links',`
+	gen_require(`
+		type udev_runtime_t;
+	')
+
+	files_search_runtime($1)
+	read_lnk_files_pattern($1, udev_runtime_t, udev_runtime_t)
```

Comment:
Based on where this is called, this doesn't seem necessary, since `udev_read_runtime_files()` already has symlink access.